typescript/web/src/pages/auth/signin.tsx

```diff
@@ -27,10 +32,25 @@ type BodyProps = {
 const Body = ({ redirectUrl }: BodyProps) => {
   return (
     <SignInProvider>
-      <VStack mt={10} mx="auto" maxW="4xl" spacing={10}>
-        {redirectUrl && <MustLogInMessage />}
-        <SignIn />
-      </VStack>
+      <Flex
+        direction="column"
+        bg={mode("gray.100", "gray.800")}
+        flexGrow={1}
+        justify="center"
+      >
+        <VStack
+          mt={10}
+          mx="auto"
```

Comment:
Why do you need to use `mx="auto"` within a flexbox container? Don't you miss an intermediary `Flex direction="row" justify="center"` + `flexGrow={1}` in this VStack?

Reply:
I don't need it, it's coming from [your last commit.](https://github.com/labelflow/labelflow/commit/3defae93f132f1e7f7f3d9022575eafaae7831f7#diff-325f4af38c2f11d76659c9fad76c9b27e3f4a9db593c0fdcb72c6eb9049ed25aR30)

Reply:
OK my bad 👍